### ISSUE TITLE
Question: Placeholders/Helpers for Jira

### ISSUE THREAD

**Issue by arkadioz** [tomasbjerre/git-changelog-maven-plugin#54]
I was looking for like a list of placeholders I could use but after some searching on your repos I found what I was looking for (but if you have more info please provide) here https://github.com/tomasbjerre/git-changelog-lib?tab=readme-ov-file#context

But I can not figure out how to use the {{urlParts}} in html template, how can I access the different positions of that?

Was trying to do something like:

`<h2>Changelog for {{repoName}} - url: {{urlParts}}[2]/{{urlParts}}[1]/{{urlParts}}[0]</h2>`

basically I just want to get the url of the repo but no success this way... I am modifying this template for Jira that I found (if there is an .md version of this one I would appreciate) https://github.com/tomasbjerre/git-changelog-lib/blob/master/examples/bitbucket_jira.html

**Comment by tomasbjerre** [comment]
Here is an example:
https://github.com/tomasbjerre/git-changelog-lib/blob/253afe33dba4195133a3fdfb924ab423f0f8599d/src/test/java/se/bjurr/gitchangelog/api/TemplatesTest.testUrlParts.approved.txt#L6
